JavaTraining/BillsBurgers/src/Main.java

```diff
@@ -0,0 +1,63 @@
+public class Main {
+
+    public static void main(String[] args) {
+
+        Burger burger = new Burger("Simple Burger", "white roll", 5);
+        System.out.println(burger.getName() + " in " + burger.getBreadType());
+        burger.setCheese(true);
+        burger.setLettuce(true);
+        burger.setTomato(true);
+//        burger.receipt();
```

Comment:
Dead code is dead... best to get into the habit of removing commented out code and debuggin println statements before pushing PRs